source/elements/oneDAL/source/common_interface/error_handling.rst

```diff
@@ -1,3 +1,79 @@
+.. _error_handling:
+
 ==============
 Error handling
 ==============
+
+Common rules
+============
+
+oneDAL error handling uses exceptions. If an error occurs, it can be propagated
+at the point of a function call. An exception will be thrown and may be caught
+by the user using standard C++ exception handling mechanisms. There are 2 ways
```

Comment:
2 should be spelled out

suggestion: add more structure when you talk about approaches (e.g., use a bullet list)